src/resmom/mom_updates_bundle.c

```diff
@@ -744,6 +744,13 @@ get_job_update(job *pjob)
 							 job_attr_def[JOB_ATR_substate].at_name, NULL, ATR_ENCODE_CLIENT, NULL);
 	}
 
+	/*
+	 * walltime must be set before encoded_used because in case of rerun
+	 * job without used walltime, the Resource_List.walltime could be used
+	 * as used.walltime for scheduling/calendaring.
```

Comment:
Since this is the mom, I wouldn't mention calendaring.  I'd just say why from a mom point of view you have to update the walltime.  Say about how if you don't update the walltime, it won't be set.

Reply:
OK, thank you.